pages/dst/index.md

```diff
@@ -17,6 +17,43 @@ vagovprod: false
 Demo page for the decision support tool. 
 </div>
 
-<script>
-  document.writeln('<p>This is a paragraph written by JavaScript.</p>');
-</script>
+        <form id="address_form">
+            <label>Street Address:</label><input type="text" name="street"/>
+            <br/>
+            <label>City:</label><input type="text" name="city"/>
+            <br/>
+            <label>State:</label><input type="text" name="state"/>
+            <br/>
+            <label>Zip:</label><input type="text" name="zip_code"/>
+            <br/>
+            <input type="submit" value="submit" />
+        </form>
+        
+        <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+        <script type="text/javascript">
+            $('#address_form').submit(function(e) {
+                    e.preventDefault();
+                    var data = {};
+                    var Form = this;
+                    $.each(this.elements, function(i, v) {
+                        var input = $(v);
+                        data[input.attr("name")] = input.val();
+                        delete data["undefined"];
+                    });
+                    $.ajax({
+                        type: 'POST',
+                        url: 'http://ec2-35-170-241-240.compute-1.amazonaws.com/',
+                        dataType: 'json',
+                        contentType: 'application/json; charset=utf-8',
+                        data: JSON.stringify(data),
+                        context: Form,
+                        success: function(callback) {
+                            console.log(callback);
+                            $(this).text('Address input was ' + callback.street + ' ' + callback.city + ' ' + callback.state + ' ' + callback.zip_code + '!');
+                        },
+                        error: function() {
+                            $(this).html("error!");
+                        }
+                    });
+                });
+        </script>
```

Comment:
I think the indentation here is causing the Markdown compiler to interpret this as a pre-formatted text, instead of raw HTML -

![image](https://user-images.githubusercontent.com/1915775/53660586-3233e480-3c2c-11e9-8398-056a291d1c39.png)

Side note - in each PR, Heroku will spawn a "review instance" so that you can see an instance of the website running with your changes. There's a `View deployment` button at the bottom of the PR where you can see this. I took this screenshot at https://vagov-content-pr-288.herokuapp.com/dst/.

Reply:
Also, if the Markdown compiler is a problem for your work, you can change the file extension of this file from `index.md` to `index.html`. Just leave the front matter (values between `---` at the top of the file) and it should still have the site layout applied.

Reply:
Yep sure enough, thanks for the catch! I'll reformat.

Reply:
@ncksllvn  Made the change, can you see if that looks right?

Reply:
Looks good!

![image](https://user-images.githubusercontent.com/1915775/53661850-6ceb4c00-3c2f-11e9-9c22-b7df21c3c923.png)